src/components/product/ProductAdminList.js

```diff
@@ -125,6 +131,14 @@ class ProductAdminList extends Component {
           renderItem={this.renderItem}
           data={this.props.products}
           keyExtractor={({ uid }) => String(uid)} deleteProduct
+          refreshControl={
```

Comment:
I think that we could use `onRefresh` and `refreshing` properties from flatlist in order to avoid import:
```
...
onRefresh={onRefresh}
refreshing={this.props.pending}
...
```
if you think that is better use the component `refreshControl` please let me know.